x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/_home.scss

```diff
@@ -28,4 +28,23 @@
   .euiToolTipAnchor {
     display: flex;
   }
-}
\ No newline at end of file
+}
+
+/*
+ * Wraps long snapshot name with ellipsis when it is rendered with an icon
+ */
+.snapshotRestorePolicyTableSnapshotFailureContainer {
+  max-width: 200px;
+  > .euiFlexItem:first-child {
```

Comment:
Do we need this empty selector?

Reply:
oops, forgot to remove that. thanks for the catch!